app/assets/stylesheets/home.scss

```diff
@@ -1,3 +1,140 @@
 // Place all the styles related to the home controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+// font-family: 'Shrikhand', cursive;
+// font-family: 'Catamaran', sans-serif;
+// font-family: 'Poiret One', cursive;
+// font-family: 'Righteous', cursive;
+
+@import 'w3';
+
+// body {
+//   font-family: 'Cabin', sans-serif;
+// }
+$teal: #009688;
+
+
+a:hover {
+  color: $teal;
+}
+
+.tag_link a {
+  text-decoration: none;
+  &:hover {
+    color: white;
+  }
+}
+
+.entry_form {
+  @extend .w3-twothird;
+  margin: 10px;
+  display: inline-block;
+}
+
+#post_title {
+  margin: 3px;
+}
+
+input:focus {
+    outline-color: $teal;
+  }
+
+.new input {
+
+   margin-left: 8px;
+  width: 25%;
+  &:focus {
+    outline-color: $teal;
+  }
+}
+
+textarea {
+  margin-left: 0px;
+  width: 50%;
+  height: 100px;
+  &:focus {
+    outline-color: orange;
+  }
+
+}
+
+label {
+  // @extend .w3-label;
+  display:block;
+  margin-left: 10px;
+}
+
+.page_header {
+  @extend .w3-left;
+  font-size: 30px;
+  margin-left: 10px;
+  height: 50px;
+  letter-spacing: 5px
+}
+
+.things {
```

Comment:
This 'things' is pretty 💩, I would recommend making it more specific so that it's readable.